EventsExpress/ClientApp/src/services/EventsExpressService.js

```diff
@@ -130,6 +131,13 @@ export default class EventsExpressService {
             : await res.json();
     }
 
+    setTwitterLoging = async (data) => {
```

Comment:
Please use `setTwitterLogin` not  `setTwitterLogin`*g*

Reply:
Typo "g" fixed.